tests/manifests/test_configs_and_mounts_consistency.py

```diff
@@ -54,6 +55,39 @@ def get_volume_from_mount(template, volume_mount):
     )
 
 
+def get_values(dict_or_list):
+    result = []
+    parsed = dict_or_list.values() if type(dict_or_list) is dict else dict_or_list
+    for v in parsed:
+        if type(v) is dict or type(v) is list:
+            result += get_values(v)
+        elif type(v) is str:
+            result.append(v)
+    return result
+
+
+def find_paths_in_contents(container, mounted_config_maps):
+    paths_found = []
+    content_to_match = [e.get("value", "") for e in container.get("env", [])]
+    content_to_match += [c for c in container.get("command", [""])[1:] + container.get("args", [])]
+    for cm in mounted_config_maps:
+        for key, content in cm["data"].items():
+            if key.endswith(".yaml"):
+                content_to_match += get_values(yaml.safe_load(content))
+
+    for content in content_to_match:
+        assert type(content) is str, f"Content must be a string: {content}"
+        for match_in in content.split("\n"):
+            for exclude in ["://", "/bin/sh", "helm.sh/"]:
+                if exclude in match_in:
+                    break
+            else:
+                for match in re.findall(r"((?<![0-9:])/[^\s\n\")`:%;,/]+[^\s\n\")`:%;,]+(?!.*noqa))", match_in):
```

Comment:
Could this have a comment as it looks like my cat walked over the keyboard in places :black_cat: